.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - run:
           name: Build Swagger Spec
           command: |
-            go get github.com/go-swagger/go-swagger/cmd/swagger
+            go install github.com/go-swagger/go-swagger/cmd/swagger@latest
```

Comment:
you don't want to pin this to a specific version?

Reply:
It wasn't a specific version before, and I didn't want to change that